src/devices/Ft232H/Ft232HGpio.cs

```diff
@@ -0,0 +1,197 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.Gpio;
+using System.Linq;
+using System.Threading;
+
+namespace Iot.Device.Ft232H
+{
+    /// <summary>
+    /// GPIO driver for the FT232H
+    /// </summary>
+    public class Ft232HGpio : GpioDriver
+    {
+        /// <summary>
+        /// Store the FTDI Device Information
+        /// </summary>
+        public Ft232HDevice DeviceInformation { get; private set; }
+
+        /// <inheritdoc/>
+        protected override int PinCount => Ft232HDevice.PinCountConst;
+
+        /// <summary>
+        /// Creates a GPIO Driver
+        /// </summary>
+        /// <param name="deviceInformation">The FT232H device</param>
+        internal Ft232HGpio(Ft232HDevice deviceInformation)
+        {
+            DeviceInformation = deviceInformation;
+            // Open device
+            DeviceInformation.GetHandle();
+        }
+
+        /// <inheritdoc/>
+        protected override int ConvertPinNumberToLogicalNumberingScheme(int pinNumber) => pinNumber;
+
+        /// <inheritdoc/>
+        protected override void OpenPin(int pinNumber)
+        {
+            if ((pinNumber < 0) || (pinNumber >= PinCount))
+            {
+                throw new ArgumentException($"Pin number can only be between 0 and {PinCount - 1}");
+            }
+
+            if (DeviceInformation.IsI2cMode && (
+                (pinNumber >= 0) && (pinNumber <= 2)))
+            {
+                throw new ArgumentException($"Can't open pin 0, 1 or 2 while I2C mode is on");
+            }
+
+            if (DeviceInformation.IsSpiMode && (
+                (pinNumber >= 0) && (pinNumber <= 2)))
+            {
+                throw new ArgumentException($"Can't open pin 0, 1 or 2 while SPI mode is on");
+            }
+
+            if (DeviceInformation._connectionSettings.Where(m => m.ChipSelectLine == pinNumber).Any())
```

Comment:
`_connectionSettings` - this variable should be called as `ConnectionSettings` because it's either internal or public